compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -387,7 +387,7 @@ impl MetaItemKind {
         tokens: &mut impl Iterator<Item = &'a TokenTree>,
     ) -> Option<MetaItemKind> {
         match tokens.next() {
-            Some(TokenTree::Delimited(_, Delimiter::Invisible, inner_tokens)) => {
+            Some(TokenTree::Delimited(_, _, Delimiter::Invisible, inner_tokens)) => {
```

Comment:
```suggestion
            Some(TokenTree::Delimited(.., Delimiter::Invisible, inner_tokens)) => {
```
`_, _, _` -> `..`
`_, _` -> `..`
In many places in this commit.